packages/admin/admin-icons/generate-icons.ts

```diff
@@ -63,12 +80,19 @@ const getFormattedText = async (text: string) => {
     return results[0].output;
 };
 
-const writeComponent = async (componentName: string, pathData: string) => {
+const writeComponent = async (icon: Icon, pathData: string) => {
     const component = await getFormattedText(`
         import { SvgIcon, SvgIconProps } from "@mui/material";
         import * as React from "react";
         
-        export default function ${componentName}(props: SvgIconProps): JSX.Element {
+        ${
+            icon.deprecated
+                ? `/**
+                    * @deprecated Do not use.
+                    */`
```

Comment:
```suggestion
                   */`
```

Reply:
The space is intentional so that the JSDoc comment has correct formatting:

<img width="267" alt="image" src="https://github.com/vivid-planet/comet/assets/48853629/22c0bb5e-3a53-4661-a431-a3142d468cd6">